ui-src/lib/support/RecordingFormatter.js

```diff
@@ -0,0 +1,70 @@
+// vim: set et sw=2:
+//
+
+import TimeFormatter from './TimeFormatter';
+
+/**
+ * Formatter for framerates
+ * @type {Intl} Formatter
+ */
+const frameRateFmt = new Intl.NumberFormat([], {
+  maximumFractionDigits: 0,
+});
+
+/**
+ * Formatter for sizes
+ * @type {Intl} Formatter
+ */
+const sizeFmt = new Intl.NumberFormat([], {
+  maximumFractionDigits: 1,
+});
+
+/**
+ * Class encapsulating formatting of recording time ranges.
+ */
+export default class RecordingFormatter {
+  /**
+   * Construct with desired time format and timezone.
+   *
+   * @param  {String} formatStr Time format string
+   * @param  {String} tz        Timezone
+   */
+  constructor(formatStr, tz) {
+    this._timeFormatter = new TimeFormatter(formatStr, tz);
+    this._singleDateStr = null;
+  }
+
+  /**
+   * Change time format string, preserving timezone.
+   *
+   * @param  {String} formatStr Time format string
+   */
+  set timeFormat(formatStr) {
+    this._timeFormatter = new TimeFormatter(formatStr, this._timeFormatter.tz);
+  }
+
+  /**
+   * [format description]
```

Comment:
fill in?

Reply:
ping. (if there's nothing interesting to say at the method level, the style guide says you can just delete the placeholder.)

Reply:
There is something interesting :-) I just merely forgot. Fixed.